pegasus/sites.v3/code.org/public/css/homepage.css

```diff
@@ -139,6 +139,106 @@ body.modal-open .supreme-container{
   }
 }
 
+#homepage .bigannounce-hoc2018 {
+  padding: 20px;
+  text-align: center;
+}
+
+@media screen and (min-width: 971px)  {
+  #homepage .bigannounce-hoc2018 {
+    padding-top: 20px;
+  }
+}
+
+#homepage .action.hoc2018 {
+  float: left;
+  margin-right: 20px;
+}
+
+#homepage .action .video-play-icon {
+  width: 24px;
+  height: 24px;
+  margin-top: -4px;
+  margin-right: 4px;
+}
+
+.hoc2018 .hero-message-line1 {
+  font-size: 24px;
+  line-height: 24px;
+  padding-bottom: 3px;
+}
+
+.hoc2018 .hero-message-line2 {
+  font-family: 'Gotham 5r', sans-serif;
+  font-size: 40px;
+  line-height: 40px;
+  padding-bottom: 40px;
+}
+
+.hoc2018 .hoc2018_highlight_word {
+  color: #00adbc;
```

Comment:
not suggesting you change, just curious why sometime this file has color word "white", sometime hex codes and sometimes rgb.  Is the one "right" way we should be aiming for?

Reply:
It's a good question.  I think we should probably standardize on `rgb` in the short term, though `black` and `white` are helpful exceptions, but in the longer term, switching to SCSS would be good.